op/status.go

```diff
@@ -472,9 +489,12 @@ func GetKubernetesClusterStatus(ctx context.Context, inf cke.Infrastructure, n *
 	return s, nil
 }
 
-func objStatus(desired, updated, available int64) bool {
-	// If we takes the status immediately after applying the resource, the value of desired may be 0.
-	// In this case, we need return false.
+func objStatus(generation, observedGeneration, desired, updated, available int64) bool {
+	if generation < observedGeneration {
```

Comment:
It should be a problem if `generation` precedes `observedGeneration`?

```suggestion
	if generation > observedGeneration {
```

Reply:
Thank you.
I fixed this.